lib/rubocop/cop/correctors/multiline_literal_brace_corrector.rb

```diff
@@ -42,8 +42,30 @@ def correct_next_line_brace(corrector)
 
         corrector.insert_before(
           last_element_range_with_trailing_comma(node).end,
+          closing_brace_content(corrector, node)
+        )
+      end
+
+      def closing_brace_content(corrector, node)
```

Comment:
I think the names `closing_brace_content` and `trailing_content_of_closing_brace` are a bit confusing, as braces can't really have content. The naming should be improved.

Reply:
@bbatsov changes done. Please review now.